docker/dockerise.sh

```diff
@@ -0,0 +1,20 @@
+#!/bin/bash
+
+# This command requires githubtoken.txt, in the repo root (it's git ingored btw). 
+# githubtoken.txt should contain github token: https://github.com/settings/tokens -> generate new token -> [x] read:packages
+
+docker build \
+   --progress plain \
+   -t testbuild \
+   --build-arg SERVER_BRANCH=309-Dockerise-local-build-of-combined-back-and-front-end \
+   --build-arg WEB_APP_BRANCH=master \
+   --build-arg NODE_VERSION=14 \
+   --build-arg POSTGRES_VERSION=12 \
+   --secret id=githubtoken,src=../githubtoken.txt \
+   .
```

Comment:
Is this `.` meant to be here?

Reply:
yeah, it tells docker build to find dockerfile in current dir